docs/guide/features.md

```diff
@@ -71,18 +71,13 @@ If a dependency doesn't work well with `"isolatedModules": true`. You can use `"
 
 - [TypeScript documentation](https://www.typescriptlang.org/tsconfig#useDefineForClassFields)
 
-Starting from Vite 2.5.0, the default value will be `true` if the TypeScript target is `ESNext` or `ES2022` or newer. It is consistent with the [behavior of `tsc` 4.3.2 and later](https://github.com/microsoft/TypeScript/pull/42663). It is also the standard ECMAScript runtime behavior.
-
+The default value will be `true` if the TypeScript target is `ES2022` or newer including `ESNext`. It is consistent with the [behavior of TypeScript 4.3.2+](https://github.com/microsoft/TypeScript/pull/42663).
 Other TypeScript targets will default to `false`.
```

Comment:
- Vite 2.5.0 is a long time ago, I think we don't need to mention it anymore.
- We don't use "tsc" in other places, so I think calling it "TypeScript" is fine.